packages/fxa-payments-server/src/components/NewUserEmailForm/index.tsx

```diff
@@ -122,21 +128,24 @@ export async function checkAccountExists(userEmail: string) {
 export async function emailInputValidationAndAccountCheck(
   value: string,
   focused: boolean,
+  setValidEmail: Function,
+  setAccountExists: Function,
   setEmailInputState: Function,
-  getString: Function,
-  checkAccountExists: Function
+  checkAccountExists: Function,
+  getString?: Function
 ) {
   let error = null;
   let valid = false;
   let hasAccount = false;
   setEmailInputState(value);
   if (isEmailValid(value)) {
     valid = true;
+    setValidEmail(value);
     const response = await checkAccountExists(value);
-    if (response.exists) {
-      hasAccount = true;
-    }
-  }
+    hasAccount = response.exists;
+    setAccountExists(response.exists);
+  } else setValidEmail('');
```

Comment:
nit: even though there's one line for the `else`, have it in a `{}` block is still preferred.